### PR TITLE
AC-454:Fixed progress bar does not show when registering patient

### PR DESCRIPTION
**Pull request by StonyMoon**
issue link: https://issues.openmrs.org/browse/AC-454

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java [hunk 1]
```diff
@@ -393,6 +393,11 @@ public void hideSoftKeys() {
 
     @Override
     public void setProgressBarVisibility(boolean visibility) {
+        if(visibility) {
+            submitConfirm.setClickable(false);
```
Comment (Sub6Resources) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java]: simplify this with one statement: `submitConfirm.setClickable(!visibility);`

Comment (StonyMoon) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/addeditpatient/AddEditPatientFragment.java]: Done.

**Comment by shivtej1505** [comment]
29495942_1346973015402181_1814726181954519040_o
29496186_1346972998735516_7716399404327895040_o
Great work @StonyMoon! 
Somewhat this PR has changed the style of the _Register Patient_ button. Compare the buttons in the two screenshop.
Could you please fix this and then this PR would be ready to be merged.

**Comment by StonyMoon** [comment]
@shivtej1505 Done. Please review. 
Seem backgroundTint does not work before Android 5.0, so I replace the background with a drawable. 
What do you think?
Thanks for your review.  :smile:

**Comment by dkayiwa** [comment]
@shivtej1505 what do you think of this?

**Pull request merged**